src/services/Store/BalanceService.tsx

```diff
@@ -103,6 +112,20 @@ const tokenBalanceFetchWrapper = async (
   }
 };
 
+const tokenBalanceMultiFetchWrapper = async (
+  provider: ProviderLike,
+  addresses: string[],
+  contractList: string[],
+  options: any
+): Promise<EthScanBalanceMap<EthScanBalanceMap<EthScanBN>>> => {
+  try {
+    const tokenBalanceMap = await getTokensBalances(provider, addresses, contractList, options);
+    return tokenBalanceMap;
+  } catch {
+    return {} as EthScanBalanceMap<EthScanBalanceMap<EthScanBN>>;
+  }
+};
```

Comment:
This simple sets the default response to to an object.
It's the responsibility of `getTokensBalances` to handle it's default value.

Reply:
Well, we changed this at one point to improve error handling: https://github.com/MyCryptoHQ/MyCrypto/commit/ee25ebe4bfcc3ba8d78ddcbb3ce50757adc37ca3

Reply:
So I would prefer to keep that.

Reply:
The referenced commit adds 12LOC for what can be done in a single line on the caller. 
```
.then(...)
.catch(_ => {} as EthScanBalanceMap<...>)
```
I just really don't see any value added by preserving an extra layer in this case. The need to add an extra function name and remember that `tokenBalanceMultiFetchWrapper === getTokensBalances || {} when Error` is a small simplification that can help us improve the code base.